rules/linux/macos_system_network_discovery.yml

```diff
@@ -0,0 +1,30 @@
+title: System Network Discovery - macOS
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection1:
+        ProcessName:
+            - '/usr/sbin/netstat'
+            - '/sbin/ifconfig'
+            - '/usr/sbin/ipconfig'
+            - '/usr/libexec/ApplicationFirewall/socketfilterfw'
+            - '/usr/sbin/networksetup'
+            - '/usr/sbin/arp'
+    selection2:
+        ProcessName: '/usr/bin/defaults'
+        Commandline|contains: 'read /Library/Preferences/com.apple.alf'
```

Comment:
could you please implement the last fix?
the thing is that both osquery and santa split arguments into a list of strings, without spaces. 
at the same time, having space in such queries makes the detection easily bypassed (just add one more space and that't it).

so I would like to ask you to change it to the following:

```
        Commandline|contains|all: 
            - 'read'
            - '/Library/Preferences/com.apple.alf'
```

The rest of the rule looks great and ready for merge. Thank you!

Reply:
The latest commit incorporates these changes. Thanks for the explanation, that really helps clarify how I'm thinking about writing these rules!